### PR TITLE
add disposableTimeout calls to disposable stores

### PR DESCRIPTION
**Pull request by joaomoreno**
Related to #193613

### CODE REVIEW

File: src/vs/base/browser/ui/list/listView.ts [hunk 1]
```diff
@@ -1544,6 +1545,7 @@ export class ListView<T> implements IListView<T> {
 			this.domNode.parentNode.removeChild(this.domNode);
 		}
 
-		dispose(this.disposables);
+		this.dragOverAnimationDisposable?.dispose();
```
Comment (Tyriar) [inline src/vs/base/browser/ui/list/listView.ts:1548]: You don't need this as it's handed via the new arg as part of `this.disposables.dispose();`

File: src/vs/workbench/contrib/terminal/browser/terminalTabsList.ts [hunk 2]
```diff
@@ -735,6 +735,8 @@ class TerminalTabsDragAndDrop implements IListDragAndDrop<ITerminalInstance> {
 		instance.focus();
 		await instance.sendPath(resource, false);
 	}
+
+	dispose(): void { }
```
Comment (Tyriar) [inline src/vs/workbench/contrib/terminal/browser/terminalTabsList.ts:739]: I think you'll have conflicts with my recently merged pr https://github.com/microsoft/vscode/pull/193666

**Review by Tyriar — CHANGES_REQUESTED** [review]

File: src/vs/base/browser/ui/list/listView.ts (hunk 1 above)
Comment (joaomoreno) [reply src/vs/base/browser/ui/list/listView.ts:1548]: This is another one: https://github.com/microsoft/vscode/blob/8162d644e7f01f9defc29ae026b724d7b0067395/src/vs/base/browser/ui/list/listView.ts#L1293-L1294

File: src/vs/workbench/contrib/terminal/browser/terminalTabsList.ts (hunk 2 above)
Comment (joaomoreno) [reply src/vs/workbench/contrib/terminal/browser/terminalTabsList.ts:743]: Don't see GH complaining about any conflicts.

Comment (Tyriar) [reply src/vs/workbench/contrib/terminal/browser/terminalTabsList.ts:743]: Sneaky conflicts 😉

**Comment by Tyriar** [comment]
Just merging won't fix it, you need to revert the changes to terminalTabsList.ts in this PR and I think the `this.disposables.add(dnd);` part in https://github.com/microsoft/vscode/pull/193666/files

**Review by Tyriar — APPROVED** [review]

**Comment by joaomoreno** [comment]
Thanks!

**Pull request merged**